### PR TITLE
Xsup 27483/Report Object References 

### PR DESCRIPTION
**Pull request by AradCarmi**
## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-27483)

## Description
STIX report indicators pulled from the feed will now contain the *Report Object References* indicator field, which holds the IDs of related indicators.

### CODE REVIEW

**Review by Ni-Knight — APPROVED** [review]
Can we do a small demo before merge? just us to make sure it works perfectly?

**Comment by AradCarmi** [comment]
> Can we do a small demo before merge? just us to make sure it works perfectly?

Yes I will schedule for the next week.

File: Packs/FeedTAXII/ReleaseNotes/1_1_33.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+
+- Added support for the **Report Object References** *Grid* indicator.
+-
```
Comment (sapirshuker) [inline Packs/FeedTAXII/ReleaseNotes/1_1_33.md:7]:
```suggestion

```

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Nice Job! please add unit-test.

File: Packs/FeedDHS/ReleaseNotes/2_0_20.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### DHS Feed v2
+
+- Added support for the **Report Object References** indicator field.
```
Comment (dantavori) [inline Packs/FeedDHS/ReleaseNotes/2_0_20.md:6]:
```suggestion
- STIX report indicators pulled from the feed will now contain the *Report Object References* indicator field, which holds the IDs of related indicators.
```
update where needed

**Review by dantavori — APPROVED** [review]
very nice! see doc note

**Pull request merged**